### PR TITLE
just kidding

### PR DESCRIPTION
**Pull request by kcubeterm**

### CODE REVIEW

File: public/app.css [hunk 1]
```diff
@@ -30,7 +30,7 @@ ul {
     margin-bottom:10px;
     margin-right: 10%;
     color: honeydew;
-    line-height:5px;
+    line-height:normal;
```
Comment (cauerego) [inline public/app.css:33]: i always prefer names rather than numbers, whenever possible!

File: public/search.js [hunk 2]
```diff
@@ -24,14 +24,20 @@ function appendData(data) {
     var search_list = ""
     for (var i = 0; i < data.hits.length; i++) {
         var div = document.createElement("div");
-        filename = data.hits[i].document.name
-        description = data.hits[i].highlights[0].snippet
         abspath = data.hits[i].document.abspath
+        filename = data.hits[i].document.name
+        if (data.hits[i].highlights.hasOwnProperty('snippet')) {
+
```
Comment (cauerego) [inline public/search.js:30]: except for this extra unnecessary white line, looks like a fine improvement... to add a sanity test.

but this kind of sanity should be automated, as soon as possible.

not even a function, something deeper. ideally an agpl library, at this point.

File: public/search.js [hunk 3]
```diff
@@ -41,9 +47,9 @@ function appendData(data) {
 
 function nextHandler() {
     page_no++;
-    document.getElementById('next').href = `/search?q=${input}&page=${page_no}`
+    document.getElementsByClassName('next').href = `/search?q=${input}&page=${page_no}`
     page_no = page_no - 2;
-    document.getElementById('previous').href = `/search?q=${input}&page=${page_no}`
+    document.getElementsByClassName('previous').href = `/search?q=${input}&page=${page_no}`
```
Comment (cauerego) [inline public/search.js:52]: little idea here, but within CSS in general, indeed using class makes more sense than id!

**Review by cauerego — COMMENTED** [review]
single comments added above.

what you mean by "just kidding"?

**Comment by kcubeterm** [comment]
Nothing, I was testing gh that's why

**Pull request merged**